src/index.html

```diff
@@ -1,12 +1,138 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Calendar</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Calendar</h1>
+  <body class="page__content">
+    <h1 class="page__title">Calendar for 2023</h1>
+
+    <label class="page__btn-lbl page__btn-lbl--january" for="1">January</label>
+    <input name="text-radio-btn" type="radio"
+    id="1" checked
+    class="page__btn-inp page__btn-inp--month--ja">
+
+    <label class="page__btn-lbl page__btn-lbl--february" for="2">February</label>
+    <input name="text-radio-btn" type="radio"
+    id="2"
+    class="page__btn-inp page__btn-inp--month--fe">
+
+    <label class="page__btn-lbl page__btn-lbl--march" for="3">March</label>
+    <input name="text-radio-btn" type="radio"
+    id="3"
+    class="page__btn-inp page__btn-inp--month--mar">
+
+    <label class="page__btn-lbl page__btn-lbl--april" for="4">April</label>
+    <input name="text-radio-btn" type="radio"
+    id="4"
+    class="page__btn-inp page__btn-inp--month--ap">
+
+    <label class="page__btn-lbl page__btn-lbl--may" for="5">May</label>
+    <input name="text-radio-btn" type="radio"
+    id="5"
+    class="page__btn-inp page__btn-inp--month--ma">
+
+    <label class="page__btn-lbl page__btn-lbl--june" for="7">June</label>
+    <input name="text-radio-btn" type="radio"
+    id="7"
+    class="page__btn-inp page__btn-inp--month--jun">
+
+    <label class="page__btn-lbl page__btn-lbl--july" for="6">July</label>
+    <input name="text-radio-btn" type="radio"
+    id="6"
+    class="page__btn-inp page__btn-inp--month--jul">
+
+    <label class="page__btn-lbl page__btn-lbl--august" for="8">August</label>
+    <input name="text-radio-btn" type="radio"
+    id="8"
+    class="page__btn-inp page__btn-inp--month--au">
+
+    <label class="page__btn-lbl page__btn-lbl--september" for="9">September</label>
+    <input name="text-radio-btn" type="radio"
+    id="9"
+    class="page__btn-inp page__btn-inp--month--se">
+
+    <label class="page__btn-lbl page__btn-lbl--october" for="10">October</label>
+    <input name="text-radio-btn" type="radio"
+    id="10"
+    class="page__btn-inp page__btn-inp--month--oc">
+
+    <label class="page__btn-lbl page__btn-lbl--november" for="11">November</label>
+    <input name="text-radio-btn" type="radio"
+    id="11"
+    class="page__btn-inp page__btn-inp--month--no">
+
+    <label class="page__btn-lbl page__btn-lbl--december" for="12">December</label>
+    <input name="text-radio-btn" type="radio"
+    id="12"
+    class="page__btn-inp page__btn-inp--month--de">
+
+    <ul class="page__numbers-block">
+      <li class="page__cell"></li>
+
+      <li class="page__cell"></li>
```

Comment:
```suggestion
      <li class="page__cell"></li>
      <li class="page__cell"></li>
```

add blank line only between multiple sibling blocks of html